WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -282,37 +315,35 @@ public void onRefreshStarted() {
                 });
     }
 
-    private @Nullable PostsListAdapter getPostListAdapter() {
+    private @NonNull PostsListAdapter getPostListAdapter() {
```

Comment:
I feel like this is a good change, but honestly I am not sure what kind of issues we might get now that we are not using the `mPostsListAdapter ` directly and making sure the adapter is always instantiated. Could you elaborate on this change a bit?